README.md

```diff
@@ -36,6 +36,20 @@ npm run dev                # Run the development server
 npm run build              # Build the entire site
 ```
 
+This node needs to be version 10.22.0, so you first have to make sure that you're entering these commands with that
+version. To do that, in your bash terminal, enter the following to install that version and use it:
+
+```bash
+nvm install 10.22.0
+```
+
+`nvm` stands for Node Version Manager, and it acts like pipenv for Python, where it allows you to switch between different `Node.js` versions. `Node.js` a runtime environment that runs websites by executing Javascript code.
+
+## Node.js References
```

Comment:
What do you think about embedding these links in the paragraph above instead of in their own separate section?